projects/ui/src/Context/AuthContext.tsx

```diff
@@ -219,3 +219,20 @@ export const AuthContextProvider = (props: AuthProviderProps) => {
     </AuthContext.Provider>
   );
 };
+
+function useIsOidcAuthLoggedIn() {
+  di(useGetCurrentUser);
+  const { data: user } = useGetCurrentUser();
+  const isOidcAuthLoggedIn = !!user?.email || !!user?.username || !!user?.name;
+  return isOidcAuthLoggedIn;
+}
+
+/**
+ * Since we support different authorization types, this is the way to tell if someone is logged in.
+ */
+export function useIsLoggedIn() {
```

Comment:
I ran into some issues keeping this in the context, related to the access token state and SWR. Splitting it out into a separate hook and using `useIsOidcAuthLoggedIn` fixed this.